frontend/src/components/HacknightsParticipants.vue

```diff
@@ -89,6 +90,11 @@ export default {
       this.$store
         .dispatch('hacknight/addParticipants', ids)
         .then(() => (this.selectedParticipants = []));
+    },
+    onDeleteParticipant(participant) {
+      const ids = [participant.id];
```

Comment:
You probably mirrored the solution for adding the participants. However @stanislawK used there `ids` in plural because there are many participants that can be added at once, but only one that can be deleted at a time given this solution we agreed upon. So please use sth like` participant_id` in singular form to name the variable and keep it consistent as you pass it to Vuex so that no one has to wonder if this is the same thing under the different names :D